### PR TITLE
some fixes when trying to process the MIND dataset

### PR DESCRIPTION
**Pull request by agramfort**
### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1985,11 +1985,20 @@ def _get_reject(
         # Automated threshold calculation requested
         import autoreject
 
+        ch_types_autoreject = list(ch_types)
+        if 'meg' in ch_types_autoreject:
+            ch_types_autoreject.remove('meg')
+            if 'mag' in epochs:
+                ch_types_autoreject.append('mag')
+            if 'grad' in epochs:
+                ch_types_autoreject.append('grad')
```
Comment (agramfort) [inline config.py:1994]: @apmellot this should fix your problem on camcan

Comment (hoechenberger) [reply config.py:1994]: Could you elaborate why this is needed? Doesn't `meg` do the right thing?

Comment (agramfort) [reply config.py:1994]: autoreject does not support "meg"

**Comment by hoechenberger** [comment]
Thanks @agramfort!

**Comment by apmellot** [comment]
@agramfort Thank you !

**Pull request merged**